the-event-loop/interval.js

```diff
@@ -0,0 +1,19 @@
+var count = 3;
+var timerId = null;
+
+function countdown() {
+  if (count === 3) {
+    console.log(count);
+    count--;
+  } else if (count === 2) {
+    console.log(count);
+    count--;
+  } else if (count === 1) {
+    console.log(count);
+    count--;
```

Comment:
Rather than check for each possible digit, you can check to make sure it's not `0` before logging Blast Off